feature-pack/src/license/full-feature-pack-licenses.xml

```diff
@@ -685,8 +685,8 @@
       </licenses>
     </dependency>
     <dependency>
-      <groupId>javax.security.enterprise</groupId>
-      <artifactId>javax.security.enterprise-api</artifactId>
+      <groupId>jakarta.security.enterprise</groupId>
+      <artifactId>jakarta.security.enterprise-api</artifactId>
```

Comment:
The dependency entries in this file are kept in alphabetical order based on groupId then artifactId. So please move this one to the correct new spot.

Reply:
Fixed.